Makefile.am

```diff
@@ -78,9 +78,6 @@ AM_CPPFLAGS  = @M0_CPPFLAGS@
 AM_CFLAGS    = @M0_CFLAGS@
 AM_LDFLAGS   = @M0_LDFLAGS@
 
```

Comment:
Need to remove extra-libs/.gitignore from Makefile.am 
- in clean-kernel-modules also extra-libs options needs to be excluded

Reply:
Done